sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RetriableWorkItem.java

```diff
@@ -45,7 +54,11 @@ private RetriableWorkItem(byte[] amqpMessage, int encodedMessageSize, int messag
     }
 
     byte[] getMessage() {
```

Comment:
it is could it be removed and instead can we create `CompositeReadableBuffer` from `encodedBytes`  or remove `RetriableWorkItem(byte[] encodedBytes, ...)` overload?

Reply:
You're right; that's the plan. I kept it hoping to ease the review while in draft mode (avoiding diff in tests, e.g.)